detectorSegmentations/src/FCCSWHCalPhiRow_k4geo.cpp

```diff
@@ -0,0 +1,676 @@
+#include "detectorSegmentations/FCCSWHCalPhiRow_k4geo.h"
+#include "DD4hep/Printout.h"
+
+namespace dd4hep {
+namespace DDSegmentation {
+
+using std::runtime_error;
+
+
+/// default constructor using an encoding string
+FCCSWHCalPhiRow_k4geo::FCCSWHCalPhiRow_k4geo(const std::string& cellEncoding) : Segmentation(cellEncoding) {
+  // define type and description
+  _type = "FCCSWHCalPhiRow_k4geo";
+  _description = "Phi-theta segmentation in the global coordinates";
+
+  // register all necessary parameters
+  registerParameter("phi_bins", "Number of bins phi", m_phiBins, 1);
+  registerParameter("offset_phi", "Angular offset in phi", m_offsetPhi, 0., SegmentationParameter::AngleUnit, true);
+  registerParameter("grid_size_row", "Cell size in row", m_gridSizeRow, std::vector<int>());
```

Comment:
Why is the grid size an integer?

Reply:
> Why is the grid size an integer?

the grid_size_row parameter controls how many rows/scintillators are combined in a cell for each layer. Only natural number of rows/scintillators can be grouped in a cell.

Reply:
so to me "cell size in row" does not sounds like a good description of this parameter.

Reply:
OK, then I will change with "Number of rows combined in a cell"